source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,188 @@
+   - [Using your own images](#using-your-own-images)
+
+## Using images from a community repository
+
+- Perform this procedure, if you want to use ready-made images by the community.
+- If you want to use your own images, skip this procedure and go to [Using your own images](#using-your-own-images) instead.
+
+**Duration**: about 30 minutes
+
+### To use images from the community
+
+1. On the **ESP32-S3-BOX** add-on, select edit.
+   - **Result**: An editor opens, showing the configuration file.
+   ![ESP32-S3-BOX config file](/images/assist/esp32-adopt-s3-01.png)
+2. For inspiration, we have prepared some images for you.
+   - Check them out in this [repository](https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/tree/main).
+3. For this tutorial, we will use some images of Frenck.
+   - Add the following lines into the `substitution` block.
+
+   {% raw %}
+
+     ```yaml
+      loading_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/loading_320_240.png
+      idle_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/idle_320_240.png
+      listening_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/listening_320_240.png
+      thinking_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/thinking_320_240.png
+      replying_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/replying_320_240.png
+      error_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/error_320_240.png
+    ```
+
+   {% endraw %}
+
+4. Save the changes and select **Install**:
+   - Depending on your environment, the installation process can take a while.
+   - Expect this to take about 25 minutes. The code needs to be compiled before it can be installed.
+![ESP32-S3-BOX config file](/images/assist/esp32-s3-config-05.png)
+   
+5. Once the installation is complete, you can see the new image on the S3-BOX.
+   - Reboot the device.
+   - Now, speak a command to test the new setting. For example, *OK Nabu turn on the light*.
+
+## Using your own images
+
+Perform this task if you want to use your own images on the S32-S3-BOX.
+
+This task consist of 2 procedures:
+
+- [Preparing your own images](#to-prepare-your-own-images)
+- [Adding your images to the configuration](#to-add-your-images-to-the-configuration)
+  
+**Duration**: ca. 35 minutes, without counting the time to create the images.
+
+### About the images
+
+To make sure the images can be displayed properly on the S3-BOX screen, check out the following requirements.
```

Comment:
We are talking about pictures so we should show example pictures to make it clear what we are talking about. Create an overview picture that includes the six pictures of our default assistant.

Reply:
Use this picture if you want:
![image](https://github.com/home-assistant/home-assistant.io/assets/5878296/da89c27e-1b34-4de3-9b85-0e39f543e146)